src/expr.rs

```diff
@@ -763,8 +769,12 @@ fn rewrite_if_else(context: &RewriteContext,
                                 offset,
                                 false)
             }
-            _ => else_block.rewrite(context, width, offset),
+            _ => {
+                last_in_chain = true;
+                else_block.rewrite(context, width, offset)
+            }
         };
+        let last_in_chain = last_in_chain;
```

Comment:
This is to remove the mutabiity of last_in_chain when it's no longer unnecessary. I am uncertain if this is desirable idiomatic Rust.